### ISSUE TITLE
unable to publish react-codemod to npm

### ISSUE THREAD

**Issue by threepointone** [reactjs/react-codemod#264]
There's a problem with npm, every time I've tried to publish the latest version, I get a 404 error (??!)
```
npm ERR! code E404
npm ERR! 404 Not Found - PUT https://registry.npmjs.org/react-codemod - Not found
npm ERR! 404 
npm ERR! 404  'react-codemod@5.2.1' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

I'm giving up at this point. If someone else with publish rights could try publishing this, that would be great. 

Of note, this is critical because it contains @lunaruan's PR https://github.com/reactjs/react-codemod/pull/262 for `update-react-import`. 

cc @cpojer, since I know he has publish rights. If either he could do a publish, or give publish rights to someone else on the react team and they  could try it themselves.

**Comment by cpojer** [comment]
Not sure what issue you ran into, maybe the registry had issues when you tried?

I published `react-codemod@5.2.1`. I don't have access to this repo to close this issue though :D

Hope you are well.

**Comment by threepointone** [comment]
Miss you bud, thanks for the assist.
